mlrun/api/utils/projects/member.py

```diff
@@ -147,3 +149,36 @@ def get_project_owner(
         name: str,
     ) -> mlrun.api.schemas.ProjectOwner:
         pass
+
+    async def post_delete_project(
```

Comment:
what uses it

Reply:
https://github.com/mlrun/mlrun/pull/3145/files#diff-04114f9c29caf76f8195786eac08c9a1ec2d55a290f2ec2bd663954f64514119R208